mw-webapp/src/model/businessModel/JobDone.ts

```diff
@@ -16,21 +16,27 @@ export class JobDone {
   public description: string;
 
   /**
-   * How long was the job done
+   * Enum @Time.unit (minute, hour, day, etc.)
    */
-  public time: Time;
+  public timeUnit: string;
 
-  constructor(jobDoneData: JobDone) {
-    this.uuid = jobDoneData.uuid;
-    this.description = jobDoneData.description;
-    this.time = new Time(jobDoneData.time.unit, jobDoneData.time.amount);
+  /**
+    * How long did the task take to complete
+    */
+  public time: number;
+
+  constructor(jobDone: JobDone) {
+    this.uuid = jobDone.uuid;
+    this.description = jobDone.description;
+    this.timeUnit = jobDone.timeUnit;
+    this.time = jobDone.time;
   }
 
   /**
    * Get formatted job that was done
    */
   public getFullJobDone() {
-    return `${this.description} (${this.time.amount} ${this.time.unit})`;
+    return `${this.description} (${this.time} ${this.timeUnit})`;
```

Comment:
does getPartialJobDone wil be implemented?

Reply:
Rename method to getJobDone